packages/realm-server/migrations/1726475945119_add-resource-created-at-to-boxel-index.js

```diff
@@ -0,0 +1,12 @@
+exports.up = (pgm) => {
+  pgm.addColumns('boxel_index', {
+    resource_created_at: { type: 'bigint' },
+  });
+
+  pgm.createIndex('boxel_index', ['resource_created_at']);
```

Comment:
Since we are ordering by `resource_created_at` and `last_modified`, I think it's a good idea to add an index for both